docs/application/dotnet/api/overview.md

```diff
@@ -1,21 +1,21 @@
 # Tizen .NET API Reference
 
-Tizen .NET provides a rich set of interfaces allowing you to build compelling TV, mobile, and wearable(preview) applications which achieve native performance. The programming environment includes the following:
+Tizen .NET provides a rich set of interfaces allowing you to build compelling TV, mobile, and wearable (preview) applications which achieve native performance. The programming environment includes the following:
 
-- **.NET Standard API**, which implements the .NET Base Class library, allows you to use the well known C# language base class libraries and features, such as collections, threading, file I/O, and LINQ.
-- **Xamarin.Forms**, which allows you to efficiently build a user interface from standard components in C# or XAML.
-- **TizenFX API**,
-   which allows you to access platform-specific features not covered by the generic .NET and Xamarin.Forms features,  such as system information  and status, battery status, sensor date, and account and connectivity services.  For the full specification, see the <a href="tizenfx/index.html" target="api">TizenFX API Reference</a>
+- **.NET Standard API** implements the .NET Base Class library, and allows you to use the well-known C# language base class libraries and features, such as collections, threading, file input/output (I/O), and Language Integrated Query (LINQ).
```

Comment:
**.NET Standard API**, which implements the .NET Base Class library, allows you ... is better.